src/CalcViewModel/Common/Automation/NarratorAnnouncement.h

```diff
@@ -30,23 +30,25 @@ public
     private:
         // Make CalculatorAnnouncement a friend class so it is the only
         // class that can access the private constructor.
-        friend class CalculatorAnnouncement;
+        // friend class CalculatorAnnouncement;
```

Comment:
Change the friend class to public as it's impossible to pass the friend class across the ABI boundary, and make the constructor internal for CalculatorAnnouncement to use. It would be better to remove the confusing comment and add extra comment for changes.

Reply:
Agree, remove the comment in the latest commit.